parboiled-core/src/main/scala-3/org/parboiled2/ParserMacros.scala

```diff
@@ -555,41 +555,49 @@ base match {
   case class StringMatch(stringTree: Expr[String]) extends OpTree {
     final private val autoExpandMaxStringLength = 8
```

Comment:
Is this feature even enabled when it's set to 0 here?

Reply:
I'm not sure I understand your question.
Let me try to reformulate it:
if we would set `autoExpandMaxStringLength` to `0`, would the optimization kick in? In the case, no because of `if s.length <= autoExpandMaxStringLength`.

It's set to `8` as in the scala 2 macros, and will be used for string constants that have a length less or equal to 8.

Reply:
Oh, wow I just misread the number, thanks for leading the blind.